templates/tweets/create.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'base.html' %}
+
+{% block title %}<title>Tweet</title>{% endblock %}
```

Comment:
```suggestion
{% block title %} Tweet {% endblock %}
```
title ブロックをオーバーライドしているので、もう一度 <title> を入れなくても大丈夫です（DeleteとDetailも同様）。